### ISSUE TITLE
MySql - Semicolons break query into subqueries

### ISSUE THREAD

**Issue by fleca** [eveningkid/denodb#128]
Hi, I am trying to add/update a mySql property (VARCHAR), but the MySQLConnector "query" method breaks the query into subqueries by splitting the query using semicolons:

I am not sure if this is an issue or if I doing something wrong.

Here is the method source code:
```
async query(
    queryDescription: QueryDescription,
    client?: MySQLClient | MySQLConnection
  ): Promise<any | any[]> {
    await this._makeConnection();

    const queryClient = client ?? this._client;
    const query = this._translator.translateToQuery(queryDescription);
    const subqueries = query.split(";");
    const queryMethod = query.toLowerCase().startsWith("select")
      ? "query"
      : "execute";

    for (let i = 0; i < subqueries.length; i++) {
      const result = await queryClient[queryMethod](subqueries[i]);

      if (i === subqueries.length - 1) {
        return result;
      }
    }
  }
```

**Comment by eveningkid** [comment]
Hi fleca,

Thanks for your question.

I am not sure I understand your issue.
Could you elaborate on "I am trying to add/update a mySql property (VARCHAR)" please?

If you could just share what you are trying to do, it will help.

Keep in touch

**Comment by fleca** [comment]
Hi eveningkid,

Ok, sorry for my bad English.
But what happens is when I try to add/update a model which contains a string field and the field value has one or more semicolons, the method "query" of the MySQLConnector class splits the query into invalid subqueries at each semicolon (const subqueries = query.split(";"))

For example I have the model Company:

```
class Company extends Model {
  static fields = {
    _id: { primaryKey: true },
    name: DataTypes.STRING,
  };

  _id!: string;
  name!: string;
}
```

Then I try to add a new record:

```
const company = new Company();
company.id= 5;
company.name= 'A ; company ; name ; containing ; semicolons';
await company.save();
```

The MySQLConnector will throw an error on the query method because it splits the CREATE query into subqueries.

**Comment by eveningkid** [comment]
Alright, I know it's certainly too late, but @stillalivx just patched this issue!

It's now available in `1.0.34` so remember to update if you're coming across this issue :)

Have a great day
